sources/SessionView.m

```diff
@@ -176,6 +178,8 @@ - (void)setBackgroundDimmed:(BOOL)backgroundDimmed {
     if (_backgroundDimmed != orig) {
         [self updateDim];
     }
+    
+    [[[self session] tab] recheckBlur];
```

Comment:
This is called from four places in `PseudoTerminal`. I'm not sure it's necessary to do this in all four cases--in fact, I believe only `_refreshTerminal` actually needs it. Can you move it up into the necessary call-sites? I'd like to keep this function true to its name.